src/concurrent/atomic64_gcc_x86_64.h

```diff
@@ -0,0 +1,77 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
do we have to provide clang version of this file for the clang compiler? FB is using clang for a lot of build process tools, including running tests.